lib/ccflay.rb

```diff
@@ -84,3 +84,25 @@ def pure_ruby_hash # :nodoc: see above
     hash
   end
 end
+
+class Sexp
+  def flatter
+    r = dup.clear
```

Comment:
These variable names aren't very clear. Can you expand them to be more descriptive?